src/db_init.h

```diff
@@ -25,7 +25,7 @@
  * version of the database? If yes, then it is a minor upgrade, if no, then it
  * is a major upgrade. In other words minor version upgrades permit downgrading
  * forked-daapd after the database was upgraded. */
-#define SCHEMA_VERSION_MAJOR 19
+#define SCHEMA_VERSION_MAJOR 20
```

Comment:
I have to say, that i am not a fan of the major/minor version for the db. In theory it would be possible to downgrade. But for that to work the user will have to change at least the db-version in the sqlite database. And it might not be possible to rerun the update on an already updated database. 

If you want, i can change it to a minor version update.